docs/apis-tools/operate-api/authentication.md

```diff
@@ -4,57 +4,141 @@ title: Authentication
 description: "Authentication requirements for accessing the Operate REST API."
 ---
 
-All Operate REST API requests require authentication.
-
-## Authentication for SaaS
-
-### Authentication via JWT access token
-
-You must pass an access token as a header in each request to the SaaS Operate API. When you create an Operate [client](/guides/setup-client-connection-credentials.md), you get all the information needed to connect to Operate.
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+All Operate REST API requests require authentication. To authenticate, generate a JWT token and pass it in each request.
+
+## Generating a token
+
+<Tabs groupId="authentication" defaultValue="saas" queryString values={
+[
+{label: 'SaaS', value: 'saas' },
+{label: 'Self-Managed', value: 'self-managed' },
+]}>
+<TabItem value='saas'>
+
+1. [Create client credentials](/guides/setup-client-connection-credentials.md) in the **Clusters > Cluster name > API** tab of [Camunda Console](https://console.camunda.io/).
+2. Add permissions to this client for **Operate**.
+3. Upon creating the client, capture the following values required to generate a token:
```

Comment:
NBC: "When" instead of "upon"?

Reply:
I actually changed this from when to upon, because it felt wrong. You see the values _as soon as you're done_ creating the client. I didn't like "after," because it implies that you can go back and get the values later (which you can't do). 

Is there a different word to use here?

Reply:
"Once"[you have created...] implies straight after
"After" has less emphasis on it being straight after
"When" implies during the process
I guess then it should be after or once, but once is a hurdle in translation as it can also mean numerically. So perhaps we can use "Once" for now, and if translation becomes a thing, we know to revisit this phrasing?

Reply:
So it could be: Once you have created the client, capture...